data/changelog.txt

```diff
@@ -192,6 +258,7 @@ Add units::temperature_delta
 Upgrade clang-tidy used in CI to LLVM 16
 Allow C++ standard includes in clang-tidy tests
 Refactor timer items to be a bit more time based
+Support for material replacement in items
```

Comment:
```suggestion
```

Reply:
I think this one is pretty important given that it's part of a request that has been around for ages.   It needs a better title though?

Reply:
Ah yeah, my mistake. For some reason I thought this was refering to a different PR.